hudi-utilities/src/test/java/org/apache/hudi/utilities/testutils/sources/AbstractDFSSourceTestBase.java

```diff
@@ -189,4 +205,63 @@ public void testReadingFromSource() throws IOException {
         Option.empty(), Long.MAX_VALUE);
     assertEquals(10101, fetch6.getBatch().get().count());
   }
+
+  @Test
+  public void testCleanUpSourceAfterCommit() throws IOException {
```

Comment:
Not sure why we have tests in this class. These classes in testutils are utility classes to assist in testing and should not have any individual tests as such. Can you move tests from this class to a separate test class. If we don't hold on this, slowly it will invite everyone to add more tests to this class.